DataConnectors/Gitlab/azuredeploy.json

```diff
@@ -0,0 +1,317 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "PlaybookName": {
+            "defaultValue": "Gitlab-Sentinel",
+            "type": "string"
+        },
+        "PersonalAccessToken": {
+            "type": "securestring",
+            "defaultValue": "<Personal Access Token>",
+            "metadata": {
+                "description": "The personal access token for gitlab instance with read access."
+            }
+        },
+        "ProjectID": {
+            "type": "string",
+            "defaultValue": "<ProjectID>",
+            "metadata": {
+                "description": "The project id for gitlab projects to be audited."
+            }
+        },
```

Comment:
This might need to be an array or we can use another api to get all the project(repo) and loop into it. In Gitlabs term project is githubs repository (https://about.gitlab.com/blog/2016/01/27/comparing-terms-gitlab-github-bitbucket/). From this logic I think this would only support 1 project(repo).

Reply:
updated to csv input